### PR TITLE
[new]  Add g2lot via single package.json

### PR DESCRIPTION
**Pull request by RaoHai**
# Pull request template for new library

Pull request for issue: #13754 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required):https://github.com/antvis/g2plot
* Official website (optional, not the repository):https://g2plot.antv.vision/
* NPM package url (optional):https://www.npmjs.com/package/@antv/g2plot
* License and its reference: MIT https://github.com/antvis/g2plot/blob/master/LICENSE
* GitHub / Bitbucket popularity (required):
  * Count of watchers:58
  * Count of stars:982
  * Count of forks:91
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week: 1048
  * Downloads in the last month:

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/g2plot/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "g2plot",
+  "filename": "dist/g2plot.js",
```
Comment (xtuc) [inline ajax/libs/g2plot/package.json]: Isn't the filename `g2plot.js` because we will move it? @MattIPv4

Comment (MattIPv4) [inline ajax/libs/g2plot/package.json]:
```suggestion
  "filename": "g2plot.js",
```
As Sven mentioned, this should not include `dist`, as the auto-update config will pull assets out of this directory into the root of the version.

File: ajax/libs/g2plot/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "version": "1.0.2",
```
Comment (MattIPv4) [inline ajax/libs/g2plot/package.json]:
```suggestion
```

Please remove the `version` attr, the auto-update bot will populate this itself when it pulls in versioned library files.

File: ajax/libs/g2plot/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "type": "git",
+    "url": "https://github.com/antvis/g2plot"
+  },
+  "license": "MIT",
+  "npmName": "@antv/g2plot",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.js"
```
Comment (MattIPv4) [inline ajax/libs/g2plot/package.json]:
```suggestion
        "**/*"
```
We might as well also include the source map, so we should remove the js extension from the glob.

**Pull request merged**